blocks/api/serializer.js

```diff
@@ -50,20 +50,28 @@ export function getSaveContent( blockType, attributes ) {
 	}
 
 	// Adding a generic classname
-	const addClassnameToElement = ( element ) => {
-		if ( ! element || ! isObject( element ) || ! className ) {
+	const addGenericAttributes = ( element ) => {
```

Comment:
Yes, I also wanted to render them separately:

![advanced-block-settings](https://user-images.githubusercontent.com/548849/29306977-2832d3e4-81a0-11e7-8c75-b26939097558.png)